recipes/obitools4/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "obitools4" %}
+{% set version = "4.4.0" %}
+{% set sha256 = "3b1bb250c300cc530aa9ecd0661a2827b8db44c58d853a94bf092492532a6609"%}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [osx]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+
+source:
+  url: "https://github.com/metabarcoding/obitools4/archive/refs/tags/Release_{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('c')}}
+    - {{ compiler('go-cgo') }}
+    - libzlib
+    - make
+    - zlib
+  run:
+    - libzlib
```

Comment:
```suggestion
```

Reply:
If you remove it, the mulled docker test fails. `ldd` shows that the binaries need it.